core/services/ocr2/plugins/ocr2vrf/coordinator/coordinator.go

```diff
@@ -45,10 +45,17 @@ const (
 
 // block is used to key into a set that tracks beacon blocks.
 type block struct {
+	hash        common.Hash
 	blockNumber uint64
 	confDelay   uint32
 }
```

Comment:
Worth noting that this type is now basically the same as `types.Block` from `ocr2vrf`. Though might be worth keeping this type separate, as the `ocr2vrf` type may change from under us.

Reply:
it's formatted in the same as `callback`, I'd like to keep it separated from that Block type for the reason you gave.